### PR TITLE
Modal popup

### PR DESCRIPTION
**Pull request by Kwaleyela-Ikafa**
in this branch we:
- Added a popup modal 
- Modal appears when the see my project button of the Multi-Post Stories card is clicked

### CODE REVIEW

**Review by sumairq — CHANGES_REQUESTED** [review]
## Hi Team, 


Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Interactive elements are working correctly
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Required Changes ♻️

- [x] In the "Works" section the cards after the first card should have a different layout that should not display the `See Project `  button. Kindly fix this issue. Cheers 👍 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
 image | image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 1]
```diff
@@ -748,6 +824,195 @@ form {
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  position: absolute;
+  width: 343px;
+  height: 718px;
+  left: calc(50% - 343px / 2);
+  top: 60px;
+}
```
Comment (vechicin) [inline styles.css:840]: On the mobile version, I can only see the modal window when I click on the first project, none of the other cards display a modal window. Please fix that.

Additionally, the modal window is not properly displayed on the page.

Screen Shot 2022-07-20 at 17 00 48

Screen Shot 2022-07-20 at 17 01 09

You need to make sure the background is blurred just like in the Figma design and that, as long as the modal window is open, the user can only see the modal window's content even if they scroll up and down the page.

File: styles.css [hunk 2]
```diff
@@ -748,6 +824,195 @@ form {
...
+  box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%), 0 6px 20px 0 rgb(0 0 0 / 19%);
+  -webkit-animation-name: animatetop;
+  -webkit-animation-duration: 0.4s;
+  animation-name: animatetop;
+  animation-duration: 0.4s;
+  background-color: #fff;
+  position: absolute;
+  width: 100%;
+  top: 15px;
+}
```
Comment (vechicin) [inline styles.css:854]: While you did address the previous reviewer's requirement, you didn't do so properly.

Screen Shot 2022-07-20 at 17 04 54

Screen Shot 2022-07-20 at 17 05 59

Please make sure your modal window positioning matches that of the Figma design. Make sure you fix the list of technologies' font as well.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Kwaleyela-Ikafa ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job implementing some of the previous reviewer's requirements.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Kwaleyela-Ikafa        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**